_data/projects.yml

```diff
@@ -159,6 +159,21 @@
   python3: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
   license: ["https://img.shields.io/badge/Good-brightgreen.svg", "Good"]
 
+- name: "EUVpy"
+  url: "https://github.com/DanBrandt/EUVpy/tree/main"
+  description: "A Python package containing several models for solar EUV irradiance."
+  logo: https://euvpy.readthedocs.io/en/latest/_images/EUVpy_logo.png
+  docs: "https://euvpy.readthedocs.io/en/latest/"
+  code: "https://github.com/DanBrandt/EUVpy/tree/main"
+  contact: "Daniel Brandt"
+  keywords: ["solar", "euv", "irradiance", "flux"]
```

Comment:
Oh sorry, our automated PR checker just ran and found `Unlisted keywords for project EUVpy: {'euv', 'flux', 'irradiance'}`

Keywords have to come from our [taxonomy](https://github.com/heliophysicsPy/heliophysicsPy.github.io/blob/main/_data/taxonomy.yml). We accept additions to that taxonomy file if you feel a keyword that isn't there should be.